app/api/games/route.tsx

```diff
@@ -1,22 +1,24 @@
-import prisma from "@/prisma/prisma";
+import prisma from "@/app/api/games/validation";
 
-export async function POST(req: Request) {
-	const body = await req.json();
+export async function POST(request: Request) {
+  const body = await request.json();
 
-	const result = await prisma.game.create({
-		data: {
-			...body,
-		},
-	});
-	if (!result)
-		return Response.json({
-			message: "error",
-			status: 500,
-		});
-	return Response.json({ message: "ok", status: 200, data: result });
+  try {
+    const result = await prisma.game.create({
+      data: {
+        ...body,
+      },
+    });
+
+    return new Response(JSON.stringify({ ...result }), { status: 201 });
+  } catch (e) {
+    return new Response(JSON.stringify({ message: "Validation error" }), {
+      status: 400,
+    });
+  }
 }
 
 export async function GET(req: Request) {
-	const result = await prisma.game.findMany();
-	return Response.json({ message: "ok", status: 200, data: result });
+  const result = await prisma.game.findMany();
+  return Response.json({ message: "ok", status: 201, data: result });
```

Comment:
status 200 was correct, [201 ](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/201) is statuses used to say that something was created, in this case nothing is being created, you're just responding with game entries :)

```suggestion
  return Response.json({ message: "ok", status: 200, data: result });
```